client/src/pages/Profile/useStyles.ts

```diff
@@ -0,0 +1,66 @@
+import { makeStyles } from '@material-ui/core/styles';
+const useStyles = makeStyles((theme) => ({
+  profileContainer: {
+    margin: theme.spacing(4),
+    height: '900px',
+    maxWidth: '760px',
+  },
+  coverImage: {
+    maxHeight: '260px',
+    width: '760px',
+    borderRadius: theme.shape.borderRadius,
+  },
+  basicInfoContainer: {
+    position: 'relative',
+    bottom: '94px',
+  },
+  profilePic: {
+    borderRadius: '50%',
+    maxWidth: '180px',
+    height: '180px',
+    width: 'auto',
+    border: '2px solid white',
+    boxShadow: '-1px 9px 11px -5px rgb(0 0 0 / 35%)',
+  },
+  locationIcon: {
+    position: 'relative',
+    top: '5px',
+  },
+  aboutContainer: {
+    margin: theme.spacing(4),
+    position: 'relative',
+    bottom: '90px',
+  },
+  description: {
+    maxWidth: '680px',
+    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(1),
+  },
+  galleryContainer: {
+    flexDirection: 'row',
+  },
+  galleryPic: {
+    margin: theme.spacing(1),
+    height: '180px',
+    width: '180px',
+  },
+  bookingContainer: {
+    margin: theme.spacing(4),
+    width: '400px',
+    height: '500px',
+  },
+  dateContainer: {
+    display: 'flex',
+    flexDirection: 'column',
```

Comment:
We can remove these two properties here - the `Grid` component has `display: 'flex'` by default and it would be cleaner to use [its `direction` prop](https://mui.com/api/grid/#props) for `flexDirection`,

Reply:
Fixed the issue..Thanks